p2p_wallet/Scenes/Main/Send/Input/SendInputCoordinator.swift

```diff
@@ -94,6 +95,7 @@ final class SendInputCoordinator: Coordinator<SendResult> {
                     self?.subject.send(.sentViaLink(link: link, transaction: model))
```

Comment:
no completion event for sentViaLink, so it would stay in Send screen, which is wrong behavior